xkit.js

```diff
@@ -872,14 +872,24 @@ XKit = {
 		},
 		get_setting: function(setting_name, default_value) {
 			try {
-				return GM_getValue(setting_name, default_value);
+				var storage_raw = GM_getValue(setting_name, default_value);
+				var storage = JSON.parse(storage_raw);
```

Comment:
I believe that if the current value is a non-JSON string this will break and return default_value incorrectly.
For example:
```javascript
set_setting("test_string", "asdf");
// Run update to this version
var x = get_setting("test_string", "default");
// x is "default", not "asdf"
```